node-require/calculate.js

```diff
@@ -0,0 +1,14 @@
+const add = require('./add');
+const subtract = require('./subtract');
+const multiply = require('./multiply');
+const divide = require('./divide');
+
+if (process.argv[3] === 'plus') {
+  console.log(add(parseInt(process.argv[2]), parseInt(process.argv[4])));
+} else if (process.argv[3] === 'minus') {
+  console.log(subtract(parseInt(process.argv[2]), parseInt(process.argv[4])));
+} else if (process.argv[3] === 'times') {
+  console.log(multiply(parseInt(process.argv[2]), parseInt(process.argv[4])));
+} else if (process.argv[3] === 'over') {
+  console.log(divide(parseInt(process.argv[2]), parseInt(process.argv[4])));
+}
```

Comment:
To avoid so much repetition it is best to save the user input into variables then use those variables for the `if` statements and passing to the math functions.

```JAVASCRIPT
const x = parseInt(process.argv[2]);
const operation = process.argv[3];
const y = parseInt(process.argv[4]);
```